scripts/publish.js

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env node
+const execSh = require('exec-sh');
+let version = require('../package.json').version;
+let name = require('../package.json').name.replace(/-/g, '_');
+ if(version.includes("rc")) {
+    version = version.replace('-', '');
+    console.log("Adjusted version to", version, "for PyPi");
+}
+ console.log(`Publishing version ${version} of ${name} to NPM & PyPi\n`);
+ console.log('>', 'python setup.py sdist');
+ execSh('git diff-index --quiet HEAD --', err => {
+    if(err) {
+        throw new Error('\nIt looks like there are uncommitted changes! Aborting until these changes have been resolved.\n');
+    } else {
+        execSh([
+                'npm publish --otp',
```

Comment:
Think you can remove the --otp, it will asks if need be.

Reply:
Okay, wasn't sure what that did, I just use `npm publish` then allow it to `401` and ask for the OTP. This was copied from the `dash-core-components` repo, so this happens there as well.